account_move_fiscalyear/views/res_config_bills_view.xml

```diff
@@ -1,22 +1,25 @@
 <odoo>
     <data>
         <record id="view_account_config_settings_inherit" model="ir.ui.view">
-            <field name="name">account.settings.inherit</field>
-            <field name="model">account.config.settings</field>
-            <field name="inherit_id" ref="account.view_account_config_settings"/>
+            <field name="name">res.settings.inherit</field>
+            <field name="model">res.config.settings</field>
+            <field name="inherit_id" ref="child_compassion.compassion_settings"/>
```

Comment:
Don't make this settings inherit from the settings of child_compassion. It's an accounting settings that should be placed in the accounting settings accordingly.